config-connector/solutions/iam/kpt/kms-key-ring/README.md

```diff
@@ -11,7 +11,14 @@ KMS Key Ring
   kpt pkg get https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git/config-connector/solutions/iam/kpt/kms-key-ring kms-key-ring
   ```
 # REQUIREMENTS
-  A working cluster with Config Connector installed and configured to manage a project with Cloud Key Management Service (KMS) API enabled.
+  -   A working Config Connector instance using the "cnrm-system" service
+      account with either `roles/cloudkms.admin` or `roles/owner` in the project
+      managed by Config Connector.
+  -   Cloud Key Management Service (KMS) API enabled in the project where Config
+      Connector is installed
+  -   Cloud Key Management Service (KMS) API enabled in the project managed by
```

Comment:
KCC can enable cloudkms.googleapis.com with ServiceUsage resource. Should we add it to the list of resources, so there are fewer requirements?

Reply:
Ack. Thank you for the suggestions, Alex. We can make it a future improvement.